docs/self-managed/modeler/web-modeler/configuration/logging.md

```diff
@@ -0,0 +1,51 @@
+---
+id: logging
+title: "Logging"
+description: "Read details on additional logging configuration for Web Modeler."
+---
+
+:::note
+Web Modeler Self-Managed is available to [enterprise customers](../../../../reference/licenses.md#web-modeler) only.
+:::
+
+## Logging configuration for the `restapi` component
+
+Web Modeler's `restapi` component uses the [logback framework](https://logback.qos.ch/) for logging. To change the
+default logging behavior, create a custom configuration file and let the `restapi` know of it by specifying the
+following environment variable:
```

Comment:
Should we maybe also mention here that it logs to the Docker container's standard output by default?